### DISCUSSION TITLE
Any step-by-step installation guide for non-docker Ubuntu server?

### DISCUSSION THREAD

**Discussion by GiladHi** [requarks/wiki#3855] [Help / Questions]
Hi,

Looking for step-by-step installation guide for Ubuntu server 20.04, but without Docker.

Since we're planning to install it on a dedicated server, I don't see the point of using Docker - I think it's  just make thing more complicated... Couldn't find a guide to do it, anyone got one?

**Comment by Phil-hostwiki** [comment]
This should work https://docs.requarks.io/install/linux

**Reply by GiladHi** [reply]
It's doesn't cover the DB installation / configuration :(

**Comment by vwlau** [comment]
Maybe you will find this link/comment helpful.

https://www.reddit.com/r/selfhosted/comments/hhmec4/simple_postgresql_setup_for_wikijs/fwdqi8t/

**Reply by GiladHi** [reply]
Thanks! it is helpful :)
